book.bib

```diff
@@ -769,3 +769,12 @@ @techreport{thorley_etal2022UpperFording
   annotation = {A Poisson Consulting and Lotic Environmental report prepared for Teck Coal Ltd., Sparwood, BC.},
   file = {/Users/airvine/Zotero/storage/SKFPH72Z/Upper-Fording-River-Westslope-Cutthroat-Trout-Population-Monitoring-2021.pdf}
 }
+
+@misc{risc2018ManualBritish,
+  title = {Manual of {{British Columbia Hydrometric Standards}}},
+  author = {{RISC}},
```

Comment:
there should be a way to define what this acronym represents in the reference and use the acronym in the citation.  Commonly, there is information on how to cite a document in the first few pages of the docuemnt to be cited. this is the case for this document
![image](https://user-images.githubusercontent.com/46538103/224107664-e20129f2-b5f5-4130-8b9e-03d94e43d553.png)